docs/proposals/202105256-cluster-class-and-managed-topologies.md

```diff
@@ -632,7 +681,8 @@ Builtin variables are available under the `builtin.` prefix. Some examples:
               the template and validating if the result is a valid YAML or JSON value.
     - We will only implement syntax validation. A semantic validation is not possible as the patches will be applied 
       to provider templates. **It’s the responsibility of the ClusterClass author to ensure the patches are semantically valid**.
-
+  - `MachineHealthCheck`
+    - (defaulting) All fields required for MachineHealthChecks, including UnhealthyConditions, are defaulted to known good values on creation.
```

Comment:
Let's drop this given that we are not defaulting on ClusterClalss creation, but we are defaulting on object creation (might be worth adding a note down below)